### PR TITLE
add rating and toast component

### PR DESCRIPTION
**Pull request by sk-ip**
review rating and toast component

### CODE REVIEW

File: component/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.bottom-right {
+    bottom: 10px;
+    right: 10px;
+}
+
+.bottom-left {
+    bottom: 10px;
+    left: 10px;
```
Comment (nyyshaaa) [inline component/toast/toast.css:41]: you can create variables for these if you are adding them as classes and planning to use at different places. Or else give them straight into the component, or is it specific to this file only? like you can do this from next time.

File: component/toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.toast-title {
+    color: hsla(0, 0%, 100%, 0.87);
+    flex-grow: 1;
+    font-size: 1rem;
+}
+
+.toast-action {
+    color: #bb86fc;
```
Comment (nyyshaaa) [inline component/toast/toast.css]: you can define variables for colors.it will be more easy to work with.

**Review by nyyshaaa — APPROVED** [review]
looks good to merge, suggested a few things which you can implemen later.

File: component/rating/rating.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/*Rating*/
+
+.checked {
+    color: orange;
```
Comment (Aditi0505) [inline component/rating/rating.css]: Please use hex values.

**Review by Aditi0505 — APPROVED** [review]

File: component/toast/toast.css (hunk 1 above)
Comment (sk-ip) [reply component/toast/toast.css:41]: thanks, I am planning to use them elsewhere as well

File: component/rating/rating.css (hunk 3 above)
Comment (sk-ip) [reply component/rating/rating.css]: thanks have made the changes

File: component/toast/toast.css (hunk 2 above)
Comment (sk-ip) [reply component/toast/toast.css]: thanks have made the changes

**Pull request merged**